package.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package packagerversion="1.10.5" version="2.0" xmlns="http://pear.php.net/dtd/package-2.0" xmlns:tasks="http://pear.php.net/dtd/tasks-1.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://pear.php.net/dtd/tasks-1.0 http://pear.php.net/dtd/tasks-1.0.xsd http://pear.php.net/dtd/package-2.0 http://pear.php.net/dtd/package-2.0.xsd">
+  <name>phalcon</name>
+  <channel>pecl.php.net</channel>
+  <summary>Phalcon is a full stack PHP framework offering low resource consumption and high performance.</summary>
+  <description>
+    Phalcon is an open source full stack framework for PHP, written as a C-extension. Phalcon is optimized for high performance. Its unique architecture allows the framework to always be memory resident, offering its functionality whenever its needed, without expensive file stats and file reads that traditional PHP frameworks employ.</description>
+  <lead>
+    <name>Ruud Boon</name>
+    <user>ruudboon</user>
+    <email>ruud@ruudboon.io</email>
+    <active>yes</active>
+  </lead>
+  <date>2020-01-11</date>
+  <time>16:00:00</time>
+  <version>
+    <release>4.0.1</release>
+    <api>4.0.1</api>
+  </version>
+  <stability>
+    <release>stable</release>
+    <api>stable</api>
+  </stability>
+  <license uri="https://license.phalcon.io">BSD 3-Clause License</license>
+  <notes>
+    Full changelog can be found at: https://github.com/phalcon/cphalcon/blob/master/CHANGELOG-4.0.md
+
+    4.0.1
+    ## Added
+
+    ## Changed
+    - Changed the logic when logging times for `Phalcon\Logger` to use `DateTimeImmutable` so as to handle microseconds if necessary. [#2893](https://github.com/phalcon/cphalcon/issues/2893)
+
+    ## Fixed
+    - Fixed `Phalcon\Mvc\Model\Criteria` Di isn't set when using `Criteria::fromInput()` [#14538](https://github.com/phalcon/cphalcon/issues/14639)
+    - Fixed `Phalcon\Db\Dialect\Mysql` removing unnecessary parentheses for `double` and `float` [#14645](https://github.com/phalcon/cphalcon/pull/14645) [@pfz](https://github.com/pfz)
+    - Fixed `Phalcon\Http\Cookie::delete` to parse the correct parameters - cannot use alternative syntax until PHP 7.3 [#14643](https://github.com/phalcon/cphalcon/issues/14643)
+    - Fixed `Phalcon\Mvc\Model::__isset` to take into account non visible properties by checking the getter if it exists [#13518](https://github.com/phalcon/cphalcon/issues/13518) [#13900](https://github.com/phalcon/cphalcon/issues/13900)
+    - Fixed `Phalcon\Mvc\Model::__set` to return a more informative message if we are tying to access a non visible property [#13518](https://github.com/phalcon/cphalcon/issues/13518) [#13900](https://github.com/phalcon/cphalcon/issues/13900)
+  </notes>
+  <contents>
+    <dir name="/">
+      <file name="config.w32" role="src" />
```

Comment:
AFAIK the right format is
```
      <file md5sum="..." name="..." role="..." />
```

I would like to provide md5sum for every file.

Reply:
This is generated by the pecl cmd. This is a template. Final output will contain this

Reply:
@sergeyklay btw. I found this out the hard way. Implemented an md5 generator at first 😅